### PR TITLE
feat(sidebar): hide terminated pci projects

### PR DESCRIPTION
**Pull request by y4nnL**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/pci-discovery`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #TAPC-395
| License          | BSD 3-Clause

- [X] Try to keep pull requests small so they can be easily reviewed.
- [X] Commits are signed-off
- [ ] Only FR translations have been updated
- [X] Branch is up-to-date with target branch
- [X] Lint has passed locally
- [X] Standalone app was ran and tested locally
- [X] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/container/src/container/common/utils.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+export type PciProject = {
+  access: string;
+  creationDate: Date;
+  description: string;
+  expiration: Date;
+  manualQuota: boolean;
+  orderId: unknown;
+  planCode: string;
+  projectName: string;
+  project_id: string;
```
Comment (dectotam) [inline packages/manager/apps/container/src/container/common/utils.ts:12]: maybe use camelCase also on projectId ?

**Review by dectotam — APPROVED** [review]

File: packages/manager/apps/container/src/container/common/utils.ts (hunk 1 above)
Comment (darsene) [reply packages/manager/apps/container/src/container/common/utils.ts:12]: @y4nnL , it's due to an API constraint ?

Comment (y4nnL) [reply packages/manager/apps/container/src/container/common/utils.ts:12]: This type is basically here to describe the API response so the compiler does not complain. If camel case is mandatory we could cast the response even though it's not ideal. You tell me

Comment (dectotam) [reply packages/manager/apps/container/src/container/common/utils.ts:12]: Ok thanks for the explanation :)

**Review by dectotam — APPROVED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Pull request closed without merging**